egs/swbd/s5c/local/chain/multi_condition/run_tdnn_7b.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
+
+set -e
+
+# configs for 'chain'
+affix=
+stage=1
+train_stage=-10
+get_egs_stage=-10
+speed_perturb=true
+dir=exp/chain/tdnn_7b  # Note: _sp will get added to this if $speed_perturb == true.
+decode_iter=
+iv_dir=exp/nnet3_rvb
+num_data_reps=1        # number of reverberated copies of data to generate
+clean_train_set=train_nodup
+
+# TDNN options
+# this script uses the new tdnn config generator so it needs a final 0 to reflect that the final layer input has no splicing
+# smoothing options
+pool_window=
```

Comment:
These pooling options were deprecated long ago-- please remove them.  When you get the final 7f script, please also 'diff' with the 7e script and see if there are any other respects in which your script is outdated-- I want it as similar as possible with 7e..  You may need to change some things and rerun.